qml/pages/ChatPage.qml

```diff
@@ -36,8 +36,11 @@ Page {
     property bool isInitialized: false;
     readonly property int myUserId: tdLibWrapper.getUserInformation().id;
     property var chatInformation;
+    property var secretChatDetails;
     property alias chatPicture: chatPictureThumbnail.photoData
     property bool isPrivateChat: false;
+    property bool isSecretChat: false;
+    property bool isSecretChatReady: false;
```

Comment:
Explicitly initialiazing properties to their default values looks weird. But consistent :)

Reply:
:)